Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -3573,23 +3675,32 @@ def main():
     Executes an integration command
     """
     command = demisto.command()
+    params = demisto.params()
     LOG(f'Command being called is {command}')
 
-    api_key = demisto.params().get('apikey')
-    api_key_id = demisto.params().get('apikey_id')
-    first_fetch_time = demisto.params().get('fetch_time', '3 days')
-    base_url = urljoin(demisto.params().get('url'), '/public_api/v1')
-    proxy = demisto.params().get('proxy')
-    verify_cert = not demisto.params().get('insecure', False)
-    statuses = demisto.params().get('status')
+    # using two different credentials object as they both fields need to be encrypted
+    api_key = params.get('apikey', {}).get('password', '')
+    api_key_id = params.get('apikey_id', {}).get('password', '')
+    if not api_key:
+        raise DemistoException('Missing API Key. Fill in a valid key in the integration configuration.')
+    if not api_key_id:
+        raise DemistoException('Missing API Key ID. Fill in a valid key ID in the integration configuration.')
```

Comment:
Redundant. If it is marked as required in the yml, It can't be empty.

Reply:
Done